### ISSUE TITLE
Failed to run quick start experiment

### ISSUE THREAD

**Issue by hitchhicker** [polyaxon/polyaxon#367]
### Describe the bug
I installed Polyaxon in minikube folloing the [tutorial](https://docs.polyaxon.com/setup/kubernetes/#install-polyaxon-on-minikube), then I tried to run the quick start experiment. After executing **polyaxon run -u**, the build was succeeded. However, we got an error below:
```
building ...
scheduled ...
starting ...
running ...
2019-02-18 17:20:45 UTC master.1 -- /bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
2019-02-18 17:20:50 UTC master.1 -- /usr/local/lib/python3.5/dist-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
2019-02-18 17:20:50 UTC master.1 --   from ._conv import register_converters as _register_converters
2019-02-18 17:20:50 UTC master.1 -- Request to http://polyaxon-polyaxon-api:80/api/v1/root/quick-start/experiments/1/ephemeraltoken/ failed with status code 405.
2019-02-18 17:20:50 UTC master.1 -- Reason:
2019-02-18 17:20:50 UTC master.1 -- Successfully downloaded train-images-idx3-ubyte.gz 9912422 bytes.
2019-02-18 17:20:50 UTC master.1 -- Extracting /data/mnist/train-images-idx3-ubyte.gz
2019-02-18 17:20:50 UTC master.1 -- Successfully downloaded train-labels-idx1-ubyte.gz 28881 bytes.
2019-02-18 17:20:50 UTC master.1 -- Extracting /data/mnist/train-labels-idx1-ubyte.gz
2019-02-18 17:20:50 UTC master.1 -- Successfully downloaded t10k-images-idx3-ubyte.gz 1648877 bytes.
2019-02-18 17:20:50 UTC master.1 -- Extracting /data/mnist/t10k-images-idx3-ubyte.gz
2019-02-18 17:20:50 UTC master.1 -- Successfully downloaded t10k-labels-idx1-ubyte.gz 4542 bytes.
2019-02-18 17:20:50 UTC master.1 -- Extracting /data/mnist/t10k-labels-idx1-ubyte.gz
2019-02-18 17:20:50 UTC master.1 -- Traceback (most recent call last):
2019-02-18 17:20:50 UTC master.1 --   File "model.py", line 139, in <module>
2019-02-18 17:20:50 UTC master.1 --     experiment = Experiment()
2019-02-18 17:20:50 UTC master.1 --   File "/usr/local/lib/python3.5/dist-packages/polyaxon_client/tracking/experiment.py", line 62, in __init__
2019-02-18 17:20:50 UTC master.1 --     persist_token=True)
2019-02-18 17:20:50 UTC master.1 --   File "/usr/local/lib/python3.5/dist-packages/polyaxon_client/api/auth.py", line 115, in login_experiment_ephemeral_token
2019-02-18 17:20:50 UTC master.1 --     AuthenticationTypes.EPHEMERAL_TOKEN, ephemeral_token)
2019-02-18 17:20:50 UTC master.1 --   File "/usr/local/lib/python3.5/dist-packages/polyaxon_client/transport/http_transport.py", line 289, in post
2019-02-18 17:20:50 UTC master.1 --     session=session)
2019-02-18 17:20:50 UTC master.1 --   File "/usr/local/lib/python3.5/dist-packages/polyaxon_client/transport/http_transport.py", line 109, in request
2019-02-18 17:20:50 UTC master.1 --     self.check_response_status(response, url)
2019-02-18 17:20:50 UTC master.1 --   File "/usr/local/lib/python3.5/dist-packages/polyaxon_client/transport/http_transport.py", line 249, in check_response_status
2019-02-18 17:20:50 UTC master.1 --     status_code=response.status_code)
2019-02-18 17:20:50 UTC master.1 -- polyaxon_client.exceptions.PolyaxonHTTPError: 405 on http://polyaxon-polyaxon-api:80/api/v1/root/quick-start/experiments/1/ephemeraltoken/.
Session ended
```

### To Reproduce

```
export POLYAXON_IP=$(minikube ip)
export POLYAXON_HTTP_PORT=$(kubectl get --namespace polyaxon -o jsonpath="{.spec.ports[0].nodePort}" services polyaxon-polyaxon-api)
export POLYAXON_WS_PORT=$(kubectl get --namespace polyaxon -o jsonpath="{.spec.ports[1].nodePort}" services polyaxon-polyaxon-api)

polyaxon config set --host=$POLYAXON_IP --http_port=$POLYAXON_HTTP_PORT  --ws_port=$POLYAXON_WS_PORT

minikube service -n polyaxon polyaxon-polyaxon-api

polyaxon login -u root -p rootpassword
polyaxon project create --name=quick-start --description='Polyaxon quick start.'
polyaxon init quick-start
polyaxon run -u
```

### Expected behavior
Experiment should succeed like what is describe in the doc
```
$ polyaxon experiment -xp 1 logs
Building -- creating image -
INFO:tensorflow:global_step/sec: 1.59552
INFO:tensorflow:loss = 0.116677, step = 101 (62.679 sec)
INFO:tensorflow:global_step/sec: 1.47121
INFO:tensorflow:loss = 0.0842577, step = 201 (67.969 sec)
... 
```
### Environment
Ubuntu 16.04
version of polyaxon: 0.3.9

**Comment by mouradmourafiq** [comment]
How are you using 0.3.9. It's not released yet?

I assume you installed `polyaxon-client==0.3.9` while running v0.3.8 polyaxon.

**Comment by mouradmourafiq** [comment]
Polyaxon-v0.3.9 just completed the release, you might have had a bad state in your cluster.

polyaxon-quick-start has been updated also updated to pin the versions used for each example, please, run:

```
helm update
```

fetch latest quick-start, and redeploy again.

Sorry for the inconvenience, I will close this issue, feel free to reopen if you still have the same problem.

**Comment by hitchhicker** [comment]
@mouradmourafiq Thank you for your quick reply. You are right, I was wrong on the version of polyaxon. It was actually polyaxon-client v0.3.8 that I used, but I fetched the lastest quick-start that was supposed to be released in v0.3.9. I will check again with the lastest polyaxon i.e. v0.3.9 that you released recently.
